app/Http/Controllers/API/v1/Quiz/QuizController.php

```diff
@@ -37,4 +39,27 @@ public function show(Request $request)
 
         return $this->showOne($quiz);
     }
-}
+
+    public function relatedQuizzes(Request $request)
+    {    
+        $relatedQuizzes = Category::join('quizzes', 'categories.id', '=', 'quizzes.category_id')
+                                ->where('categories.id', $request->category_id)
+                                ->where('quizzes.id', '!=', $request->quiz_id )
+                                ->limit(4)
+                                ->get();
+        
+        $relatedQuizzesId = $relatedQuizzes->pluck('quiz_id')->all();
+
+        $attempts = QuizTaken::whereIn('quiz_id', $relatedQuizzesId)
+                               ->where('user_id', Auth::user()->id)
+                               ->orderBy('quiz_id', 'asc')
+                               ->get();
```

Comment:
From now on, let us use one level indentation only.
Will probably create separate tasks for other files.

```suggestion
        $relatedQuizzes = Category::join('quizzes', 'categories.id', '=', 'quizzes.category_id')
            ->where('categories.id', $request->category_id)
            ->where('quizzes.id', '!=', $request->quiz_id )
            ->limit(4)
            ->get();
        
        $relatedQuizzesId = $relatedQuizzes->pluck('quiz_id')->all();

        $attempts = QuizTaken::whereIn('quiz_id', $relatedQuizzesId)
           ->where('user_id', Auth::user()->id)
           ->orderBy('quiz_id', 'asc')
           ->get();
```